### DISCUSSION TITLE
Need Sanitize HTML option back in Table Panel

### DISCUSSION THREAD

**Discussion by GollaSaiVenkatesh** [grafana/grafana#55663] [Dashboards, Panels & Visualizations]
The old table panel's sanitize html option is so good . can you guys please try to include that option back in grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
